.github/workflows/build.yml

```diff
@@ -41,23 +43,40 @@ jobs:
         run: |
           /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
           brew install zig make
+      # TODO: msys2 doesn't have zig pkg
+      - name: Install zig on windows
+        if: inputs.platform == 'windows'
+        uses: mlugg/setup-zig@v1
```

Comment:
I would really not use some random unofficial github action

Reply:
I haven't found a simple way to install zig. The zig package of msys2 cannot work. Or I can manually download zig through the command line and add it to the global environment variable, which is a bit troublesome.

Reply:
Do we need msys2? If there is no alternative we can use a smaller GHA, but use a commit hash not a tag. And we need to make sure it doesnt do funny business.

Reply:
When there are c projects as dependencies, msys2 is currently the best solution, because projects like rquickjs rely on some linux tools, such as patch, perl, etc.
setup-msys2 does not seem to provide a version lock config, we can only trust it like we trust npm that it will not do funny business

Reply:
This would install zig if running on powershell:
https://github.com/awslabs/llrt/blob/a80dfe08f2dd8071723cfa385757aa4958f3023c/.github/workflows/build.yml#L44

Reply:
Ok, we need to add one more environment variable, Let's see if it works
```
    export PATH="$PATH:/c/Users/$USER/scoop/shims"
```